docs/experiments-msmarco-doc.md

```diff
@@ -126,3 +126,4 @@ As expected, BM25 tuning makes a big difference!
 + Results replicated by [@stariqmi](https://github.com/stariqmi) on 2020-05-28 (commit [`4914305`](https://github.com/castorini/anserini/commit/455169ea6a09f637817a6c4b4f6837dcc845f5f7))
 + Results replicated by [@justinborromeo](https://github.com/justinborromeo) on 2020-06-11 (commit[`7954eab`](https://github.com/castorini/anserini/commit/7954eab43f17bb8d254987d5873933c0b9596bb4))
 + Results replicated by [@yxzhu16](https://github.com/yxzhu16) on 2020-07-03 (commit [`68ace26`](https://github.com/castorini/anserini/commit/68ace26d0418a769df3d2b21e946495e54d462f6))
++ Results replicated by [@LizzyZhang-tutu](https://github.com/LizzyZhang-tutu) on 2020-07-13 (commit [`8c98d5b`](https://github.com/LizzyZhang-tutu/anserini/commit/8c98d5ba0795bbea01bcef1e21abb153fe4c3da1))
```

Comment:
can you change the commit URL to refer to the source in castorini, not your local fork?

Reply:
updated to source URL.